src/service/token-service.ts

```
@@ -1,25 +1,41 @@
-    if (!payload) return null
-    return JSON.parse(decodeBase64Url(payload)) as Record<string, unknown>
-  } catch {
-    return null
-  }
+function updateSnapshot() {
+  tokensSnapshot = { ...tokensSnapshot, accessToken: getAccessToken(), refreshToken: getRefreshToken() }
+}
+
+function notifyListeners() {
+  updateSnapshot()
+  listeners.forEach((callback) => callback())
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)